README.md

```diff
@@ -78,39 +78,39 @@ To create the database tables ...
    npm run db:migrate
    ```
 
-#### Emails
+#### Trigger Breach Alert Email
+Breach alert emails are triggered via HIBP which, for dev purposes, is set up to work with a [Mailinator](https://www.mailinator.com) email account.
 
-The included `.env-dist` sets `DEBUG_DUMMY_SMTP=1` which disables emails.
+To set up your environment for email testing with Mailinator:
+1. In your .env file, confirm or add values for `SMTP_URL`, `EMAIL_FROM`, `HIBP_KANON_API_TOKEN`, and `HIBP_API_TOKEN`
 
-To send emails, you'll need to unset `DEBUG_DUMMY_SMTP` and supply real SMTP
-config values for sending email.
+2. If you don't have a local FxA account, sign up on localhost.  You'll need to ensure `FXA_ENABLED=true` and confirm/add the value for `OAUTH_CLIENT_SECRET` in your .env file.
 
-You can set and source these via the `.env` file, or set them directly:
+3. Start/restart your server
 
-```sh
-export DEBUG_DUMMY_SMTP=
-export SMTP_HOST=<your-smtp-host>
-export SMTP_PORT=<your-smtp-port>
-export SMTP_USERNAME=<your-username>
-export SMTP_PASSWORD=<your-password>
-```
+4. Login to your Monitor account in-browser, and add localmonitor20200827@mailinator.com to list of monitored email addresses
 
-##### Trigger a breach alert email
-To trigger a breach alert email, you need to make a `POST /hibp/notify` request:
+5. In your Monitor settings (in-browser), make sure notification preferences specify "Send breach alerts to the affected email address" (should be default).  This will send the alert to the Mailinator account.
```

Comment:
```suggestion
5. In your [Monitor settings](http://localhost:6060/user/dashboard), make sure notification preferences specify "Send breach alerts to the affected email address" (should be default).  This will send the alert to the Mailinator account.
```